src/bundle/Command/CronRunCommand.php

```diff
@@ -46,6 +47,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $cron->setExecutor(new Executor());
         $cron->setResolver($resolver);
 
-        $cron->run();
+        $reports = $cron->run();
+
+        while ($cron->isRunning()) {}
```

Comment:
what is the purpose of this?

Reply:
@alongosz  The reports output is not available before the process has finished.